test/system/005-info.bats

```diff
@@ -53,4 +53,8 @@ store.imageStore.number   | 1
 
 }
 
+@test "podman info --storage-opt="" " {
```

Comment:
Suggestion: change "" (doublequote doublequote) to '' (singlequote singlequote) for log readability.

Reply:
Fixed